### ISSUE TITLE
Review and update threshold listings in extended output

### ISSUE THREAD

**Issue by atc0005** [atc0005/check-vmware#119]
While working on #66 the language used for that plugin and the other snapshot plugins stood out:

- snapshots age
  - `CRITICAL: 2 day old snapshots present`
  - `WARNING: 1 day old snapshots present`
- snapshots size
  - `CRITICAL: snapshots of 50 GB (combined size) present`
  - `WARNING: snapshots of 30 GB (combined size) present`

These are *thresholds*, and the description should clearly indicate that. For example, the **present** word above makes it sound like having a 1 day old snapshot is enough to trigger a `WARNING` state (if specifying 1 day), but it's not, that is the *threshold*. The same for the 30 GB snapshot. Both scenarios are not enough to trigger a `WARNING` state.

Once the values (age, size) go *past* the threshold is when the state changes.

In short, the `present` word will need to go. I'll also need to review the other threshold statements to make sure they're accurate.

**Comment by atc0005** [comment]
Some plugins will likely retain threshold descriptions which are technically not *thresholds* per se. For example:

- `check_vmware_vhw`
  - `Non-homogenous hardware versions`
    - it's hard to have values on both sides of that state; this is more of a boolean state
- `check_vmware_tools`: ``
  - `Tools not running (e.g., crashed, low memory scenario) or installed.`
    - same, hard to be on either side of this

For the other plugins, it should be a little easier to reword the details to add clarity.
